ltm/EntityMetadata meta

# static fields
string type
string name
geometry_msgs/Point size
sensor_msgs/Image image

# dynamic fields
bool clean
string map_name
geometry_msgs/Pose location   # location (/map frame)